modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/GcalControllerEpics.scala

```diff
@@ -22,6 +22,7 @@ object GcalControllerEpics extends GcalController {
       v match {
         case BinaryOnOff.OFF => LampState.Off
         case BinaryOnOff.ON  => LampState.On
+        case _               => sys.error("Cannot happen")
```

Comment:
The error produced is

```
[error] It would fail on the following input: (x: edu.gemini.seqexec.server.gcal.BinaryOnOff forSome x not in (OFF, ON))
[error]       v match {
```

I'm not using those compiler options yet. I'll harmonize them when we move to 2.12

At the moment we have:

```scala
    val scalacOptions = Seq(
      "-unchecked",
      "-deprecation",
      "-encoding", "UTF-8", // Keep on the same line
      "-feature",
      "-language:existentials",
      "-language:higherKinds",
      "-language:implicitConversions",
      "-target:jvm-1.8",
      "-Xlint",
      "-Xlint:-stars-align",
      "-Xfatal-warnings",
      "-Ywarn-unused-import"
    )
```